images/metering-ansible-operator/roles/meteringconfig/tasks/configure_metastore.yml

```diff
@@ -0,0 +1,10 @@
+- name: Get metastore secret
+  block:
+    - name: Get Metastore storage credentials secret
+      k8s_info:
+        api_version: v1
+        kind: Secret
+        name: "{{ meteringconfig_spec.hive.spec.config.db.secretName }}"
+        namespace: "{{ meta.namespace }}"
+      register: metastore_secret
+      when: meteringconfig_spec.hive.spec.config.db is define and meteringconfig_spec.hive.spec.config.db.secretName | length > 0
```

Comment:
```suggestion
      when: meteringconfig_spec.hive.spec.config.db is defined and meteringconfig_spec.hive.spec.config.db.secretName | length > 0
```

Reply:
This conditional should be moved to the block-level and not the task-level like it's currently configured.

Reply:
We should also use `meteringconfig_spec_overrides` instead of `meteringconfig_spec`.